packages/tokens/README.md

```diff
@@ -135,14 +135,17 @@ console.log(div.style.backgroundColor);
 
 Our variables are also available as scss variables that make it possible to consume them in sass based projects.
 
+> Note that the SASS compiler does not ship with built in support for `node_modules`. You will either have to use relative paths or use a loader like [node-sass-package-importer](https://www.npmjs.com/package/node-sass-package-importer) or [sass-module-importer](https://www.npmjs.com/package/sass-module-importer).
+> Configuration for this is NOT part of this guide.
+
 ```scss
 // Import the design tokens first.
 // This can be done in a sass file or in any other way described above.
 // Make sure to NOT add the .css file suffix, this will not work in sass
-@import "~@synergy-design-system/tokens/themes/light";
+@import "./node_modules/@synergy-design-system/tokens/themes/light";
 
 // Import the scss variables
-@import "@synergy-design-system/tokens/scss/tokens";
+@import "./node_modules/@synergy-design-system/tokens/dist/scss/tokens";
```

Comment:
Maybe we should add documentation for both import ways. For relative import and the other one. Because here again on relative import the **dist** is needed and on non-relative imports there must be **no dist**.

![image](https://github.com/synergy-design-system/synergy-design-system/assets/115139104/f8d9d618-fead-48e5-b349-9d4b2f212b75)

Reply:
I have opted to remove the references to the non relative ways of importing completely. This was done as there are multiple loaders for sass in the various build pipelines with different configurations. There seems to be no "one fits it all" approach there. Please have a look at the updated documentation and provide feedback about it.

Reply:
I like the updated documentation